pyproject.toml

```diff
@@ -1,8 +1,19 @@
 [tool.poetry]
 name = "mognet"
-version = "1.3.1"
-description = ""
+version = "1.3.2"
+description = "Mognet is a fast, simple framework to build distributed applications using task queues."
 authors = ["André Carvalho <afecarvalho@gmail.com>", "Christoph Auer <cau@zurich.ibm.com>", "Michele Dolfi <dol@zurich.ibm.com>", "Peter Staar <taa@zurich.ibm.com>"]
+license = "MIT"
+readme = "README.md"
+homepage = "https://ibm.github.io/project-mognet/"
+repository = "https://github.com/IBM/project-mognet"
+documentation = "https://ibm.github.io/project-mognet/"
+classifiers = [
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Software Development :: Version Control :: Git",
+    "Development Status :: 5 - Production/Stable",
+    "Typing :: Typed"
```

Comment:
I think we should add a `py.typed` file for MyPy. See https://mypy.readthedocs.io/en/stable/installed_packages.html#creating-pep-561-compatible-packages

Reply:
just an empty file there?

Reply:
Done!